starlite/parsers.py

```diff
@@ -35,19 +35,37 @@ def parse_query_string(query_string: bytes, encoding: str = "utf-8") -> Tuple[Tu
         query_string: A query string.
         encoding: The encoding to use.
 
+    Returns:
+        A tuple of key value pairs.
+    """
+    _bools = {b"true": True, b"false": False, b"True": True, b"False": False}
+    return tuple(
+        (k.decode(encoding), v.decode(encoding) if v not in _bools else _bools[v])
+        for k, v in parse_qsl(query_string, keep_blank_values=True)
+    )
+
+
+@lru_cache(1024)
+def parse_form_query_string(form_query_string: bytes, encoding: str = "utf-8") -> Tuple[Tuple[str, Any], ...]:
+    """Parse a form query string into a tuple of key value pairs.
+
+    Args:
+        form_query_string: A query string.
+        encoding: The encoding to use.
+
     Returns:
         A tuple of key value pairs.
     """
     _bools = {"true": True, "false": False, "True": True, "False": False}
     return tuple(
         (k, v if v not in _bools else _bools[v])
-        for k, v in parse_qsl(str(query_string, encoding=encoding), keep_blank_values=True, encoding=encoding)
+        for k, v in parse_qsl(str(form_query_string, encoding=encoding), keep_blank_values=True, encoding=encoding)
```

Comment:
why do you need to pass encoding to `str`, and then again to `parse_qsl`?

Reply:
Good question. That's a mistake on my side.
I'll remove the encoding from `parse_qsl`